tools/retest/main.go

```diff
@@ -93,7 +94,11 @@ issues:
 			continue
 		}
 		log.Printf("#%d jobs to retest: %s", prNumber, strings.Join(jobsToRetest, ", "))
-		newComments := commentsToCreate(statuses, jobsToRetest, shouldRetestFailedStatuses(statuses, userComments))
+		shouldRetestErr := shouldRetestFailedStatuses(statuses, userComments)
```

Comment:
Now that this no longer returns a `bool`, I think a better name would be something like:
```suggestion
		shouldRetestErr := getSkipRetestReason(statuses, userComments)
```
I'm also not sure `error` is the right type here, perhaps a `string` would suffice. Up to you.

Reply:
Hmm, this rename would be a bit confusing, cause that function checks only one potential skip reason, whereas other reasons are checked in different places. I could try to refactor it more... Let me see what I can do.